### ISSUE TITLE
[Feature] Add option to select file on server as scene cover image

### ISSUE THREAD

**Issue by timo95** [stashapp/stash#3422]
**Is your feature request related to a problem? Please describe.**
Many downloads include a cover image in the same folder as the video file.
The `Set image` button only allows to choose local files (`from file`), not server files. -> I can't select these covers without mounting the server data locally and navigating to the files.

**Describe the solution you'd like**
Add `from remote file` option to the `Set image` button (and/or rename `from file` to `from local file`).
Open a file browser to the primary video files folder.
Optional: Some easy way to open the file browser to any non primary video file.

**Describe alternatives you've considered**
Use a scraper to automatically add cover images with the same name as the video file. This doesn't work as good with different names and subfolders though.

**Comment by puc9** [comment]
I would also like this, but I have a workaround for you:
You could copy the stash URL to the image you want to use as cover (it works for everything else to set as cover) and use it as a URL not a file. Right click and copy link to image works as well.

**Comment by timo95** [comment]
I have been doing that for some tag images. Except, that i don't want to scan dvd covers into stash just to assign them as covers and remove them again in the end. Though it is a viable workaround if you scan them anyways.

**Comment by puc9** [comment]
That's fair and as I said, it's just an workaround, I do think that the feature would be useful

**Comment by LeGrosFromage** [comment]
Can you not copy the image to the clipboard then do a CTRL + V when you're editing?  I do that all the time.

**Comment by 81hvac1k02** [comment]
I would like to see this too. 
And to be able to set photo for performer by choosing from photos they're tagged in.

**Comment by zkczkczkc** [comment]
Yes, I really need this function.I usually store the cover file in the same directory as the video and give it the same name as the video (for example, XXX.mp4 and XXX.jpg).I suggest that when a video file contains a duplicate image file with the same name or an image file named "cover" or similar keywords, it should automatically be set as the scene cover.

**Comment by DogmaDragon** [comment]
https://discourse.stashapp.cc/t/cover-from-file/5113
